apps/modernization-ui/src/apps/page-builder/pages/EditPage/EditPageTabs/EditPageTabs.tsx

```diff
@@ -0,0 +1,179 @@
+                        <h4>Manage tabs</h4>
+                    </div>
+                </ModalToggleButton>
+            </div>
+            <ModalComponent
+                size={'tall'}
+                modalRef={modalRef}
+                modalHeading={
+                    !isAdding ? (
+                        <div className="manage-tabs-header">
+                            <div>Manage tabs</div>
+                            <Button className="" type="button" onClick={() => setIsAdding(true)}>
+                                <Icon.Add className="margin-right-05em add-tab-icon" />
+                                <span>Add new tab</span>
+                            </Button>
+                        </div>
+                    ) : (
+                        'Manage tabs'
+                    )
+                }
+                modalBody={
+                    <div className="edit-page-tabs__modal--body">
+                        {addSuccess ? (
+                            <AlertBanner type="success">You've successfully added a new tab!</AlertBanner>
+                        ) : editSuccess ? (
+                            <AlertBanner type="success" onClose={() => resetEditPageTabs()}>
+                                <p>
+                                    You've successfully saved your changes to <span>&nbsp;{tabDetails?.name}!</span>
+                                </p>
+                            </AlertBanner>
+                        ) : null}
+                        {isAdding ? (
+                            <AddEditTab setTabDetails={setTabDetails} />
+                        ) : isEditing ? (
+                            <AddEditTab tabData={tabs[selectedTabIndex!]} setTabDetails={setTabDetails} />
+                        ) : (
+                            <ManageTabs tabs={tabs} setSelectedEditTab={handleEditTab} />
+                        )}
+                    </div>
+                }
+                modalFooter={
+                    isAdding ? (
+                        <div className="margin-bottom-1em add-tab-modal ds-u-text-align--right ">
+                            <Button className="submit-btn" onClick={handleAddTab} type="button">
+                                Add tab
+                            </Button>
+                            <ModalToggleButton modalRef={modalRef} outline onClick={() => setIsAdding(false)} closer>
+                                Cancel
+                            </ModalToggleButton>
+                        </div>
+                    ) : isEditing ? (
+                        <>
+                            <Button type="button" onClick={() => handleUpdateTab()}>
+                                Save
+                            </Button>
+                            <ModalToggleButton modalRef={modalRef} outline onClick={() => setIsEditing(false)} closer>
+                                Cancel
+                            </ModalToggleButton>
```

Comment:
Question: If you are editing a tab and click `Cancel`

![image](https://github.com/CDCgov/NEDSS-Modernization/assets/109251240/358ab251-dda6-4fa2-a140-0f75406c6065)


It currently closes the modal completely. Should this "cancel" the edit mode and go back to the default edit state:
![image](https://github.com/CDCgov/NEDSS-Modernization/assets/109251240/5be5a7b8-1dc8-4095-8aef-ef34827152ab)